README.md

```diff
@@ -1,6 +1,7 @@
 # eval
 
 ![Lint](https://github.com/instructlab/eval/actions/workflows/lint.yml/badge.svg?branch=main)
+![Test](https://github.com/instructlab/eval/actions/workflows/test.yml/badge.svg?branch=main)
```

Comment:
@nathan-weinberg on line 38, 39 of this file can you mention how to run the tests with pytest individually and as a group?

Reply:
Yes!